datadog-iam-user-handler/resource-role.yaml

```diff
@@ -0,0 +1,31 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
I don't think we need this role for any of the resources. Let's ignore it.

Reply:
Agreed, I think this would only be for interacting with AWS resources?

Reply:
Ok, I'll remove all `resource-role.yaml` files and add them to gitignore.